### PR TITLE
45302 move simplified

### PR DESCRIPTION
**Pull request by rth-axelor**

### CODE REVIEW

File: axelor-account/src/main/resources/views/MoveLine.xml [hunk 1]
```diff
@@ -187,6 +187,43 @@
...
+    <field name="taxLine" hidden="true" readonlyIf="!account.isTaxAuthorizedOnMoveLine"
+      domain="self.endDate > :date OR self.endDate IS NULL"
+      requiredIf="account.isTaxRequiredOnMoveLine || account.accountType.technicalTypeSelect == 'income' || account.accountType.technicalTypeSelect == 'charge' || account.accountType.technicalTypeSelect == 'tax' || account.accountType.technicalTypeSelect == 'immobilisation'"/>
+    <field name="origin" hidden="true" width="120"/>
+    <field name="description"
+      requiredIf="$isDescriptionRequired || move.company.accountConfig.isDescriptionRequired"/>
+    <field name="reconcileGroup" hidden="true" width="80"/>
+    <field name="$isDescriptionRequired" hidden="true" type="boolean"/>
+    <field name="move.company.accountConfig.isDescriptionRequired" hidden="true"/>
+    <field name="isOtherCurrency" hidden="true"/>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/MoveLine.xml:224]: Please put all hidden fields at the end (or remove if unnecessary) and remove their other attributes

File: axelor-account/src/main/resources/views/Move.xml [hunk 2]
```diff
@@ -275,6 +275,202 @@
...
+        <field name="companyCurrency" readonly="true"/>
+        <field name="currencyCode" hidden="true"/>
+      </panel>
+      <panel name="othersPanel" hidden="true" title="Others">
+        <field name="ignoreInAccountingOk" readonly="true" showIf="ignoreInAccountingOk"/>
+        <field name="ignoreInDebtRecoveryOk"/>
+        <field name="rejectOk" readonly="true" showIf="rejectOk"/>
+        <field name="attrs"/>
+      </panel>
+    </panel-tabs>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:467]: All panels from panel tab are hidden so remove if unnecessary

File: axelor-account/src/main/resources/views/Move.xml [hunk 3]
```diff
@@ -275,6 +275,202 @@
...
+	                     <span class="label label-info" style="margin: 5px 0 !important; display: inline-table; line-height: initial;" ng-show="record.technicalOriginSelect == 2" x-translate>Automatic</span>
+	                     <span class="label label-info" style="margin: 5px 0 !important; display: inline-table; line-height: initial;" ng-show="record.technicalOriginSelect == 3" x-translate>Template</span>
+	                     <span class="label label-info" style="margin: 5px 0 !important; display: inline-table; line-height: initial;" ng-show="record.technicalOriginSelect == 4" x-translate>Import</span>
+	                    </h4>
+
+	                ]]></viewer>
+        </field>
+        <field name="getInfoFromFirstMoveLineOk" hidden="true" colSpan="12"
+          widget="boolean-switch"/>
+        <field name="journal.authorizeSimulatedMove" hidden="true"/>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:359]: Remove hidden fields if unnecessary

File: axelor-account/src/main/resources/views/Move.xml [hunk 4]
```diff
@@ -275,6 +275,202 @@
...
+      <button name="deleteMoveBtn" title="delete" showIf="id != null" icon="fa-trash"
+        onClick="action-move-method-delete-move"/>
+    </toolbar>
+    <panel name="mainPanel" colSpan="12">
+      <panel name="moveDetailsPanel" colSpan="9">
+        <field name="reference" colSpan="12"/>
+        <panel name="moveContentPanel"
+          readonlyIf="moveLineList.length > 0 &amp;&amp; statusSelect == 3" colSpan="12">
+          <field name="company" widget="SuggestBox" canEdit="false"
+            onChange="action-move-record-journal,action-move-method-get-period,action-move-attrs-validate-button-title,action-method-hide-axis-analytic-account-move-line"
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:297]: Please put actions in action-group

File: axelor-account/src/main/resources/views/Move.xml [hunk 5]
```diff
@@ -275,6 +275,202 @@
...
+            form-view="company-form" grid-view="company-grid"/>
+          <field name="journal" colSpan="3" canNew="false" canEdit="false"
+            onChange="action-move-account-record-set-journal-technical-type-select"
+            domain="self.statusSelect = 1 AND self.company = :company" form-view="journal-form"
+            grid-view="journal-grid"/>
+          <field name="date" hideIf="statusSelect == 3" colSpan="3"
+            onChange="action-move-method-get-period"/>
+          <field hidden="true" name="period" canView="false" canEdit="false"
+            form-view="period-form" grid-view="period-grid"/>
+          <field name="originDate" hidden="true" colSpan="3"/>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:307]: Please remove hidden fields if unnecessary

File: axelor-account/src/main/resources/views/Move.xml [hunk 6]
```diff
@@ -275,6 +275,202 @@
...
+          <field if="__config__.app.getApp('base').getEnableTradingNamesManagement()"
+            name="tradingName" canNew="false" onSelect="action-move-set-trading-name-domain"
+            form-view="trading-name-form" grid-view="trading-name-grid"/>
+          <field name="partner" hidden="true" colSpan="6" canNew="false" canEdit="false"
+            onChange="action-account-move-record-set-currency"
+            onSelect="action-account-move-method-filter-partner" form-view="partner-form"
+            grid-view="partner-grid"/>
+          <field hidden="true" name="currency" colSpan="3"
+            onChange="action-move-record-set-currency-code"/>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:316]: Please remove hidden fields if unnecessary

File: axelor-account/src/main/resources/views/Move.xml [hunk 7]
```diff
@@ -275,6 +275,202 @@
...
+          <field name="paymentMode" colSpan="3"/>
+          <field name="$journalTechnicalTypeSelect" hidden="true" type="integer"/>
```
Comment (aol-axelor) [inline axelor-account/src/main/resources/views/Move.xml:318]: Please remove hidden field if unnecessary

**Review by ale-axelor — CHANGES_REQUESTED** [review]

**Pull request merged**